noop.go

```diff
@@ -20,7 +20,7 @@ package go2sky
 import (
 	"time"
 
-	v3 "github.com/SkyAPM/go2sky/reporter/grpc/language-agent"
+	v3 "skywalking.apache.org/repo/goapi/collect/language/agent/v3"
```

Comment:
Redundant alias

Reply:
I tried to remove `v3` alias, nothing wrong happens. @mrproliu Could you recheck what happens?

Reply:
![image](https://user-images.githubusercontent.com/3417650/113575099-f0b66480-964f-11eb-8f3b-bf5efda4269d.png)

Hi, here is the error that I got. It notices me the files are not `goimports`, is there did I use it wrong?

Reply:
Seems your golangci-lint issue. I can see from my logs, everything is fine.

Reply:
You could try to submit a version without alias, and let CI to recheck my result for you.

Reply:
I have to try to use `goimport` command on the `noop.go` files. There is the command `goimports noop.go`, and it output the same result(include the `v3` alias), is I miss anything? Also, I found [an issue](https://github.com/golang/go/issues/30051), it seems like the redundant alias is tight. 
![image](https://user-images.githubusercontent.com/3417650/113576635-7e934f00-9652-11eb-8d52-73bfd7a8e76e.png)